bpf/cpu/cpu.bpf.c

```diff
@@ -747,9 +747,18 @@ int walk_user_stacktrace_impl(struct bpf_perf_event_data *ctx) {
         return 1;
       }
 
-      // TODO(sylfrena): add comments to explain calculations
+      // Stacktraces are saved return addresses from function calls pushed onto a stack
+      // The base pointer (`rbp` in x86_64) is first register pushed onto the stack and holds the beginning of the stack
```

Comment:
Sorry, just noticed this now... is it the first one? :D Perhaps we can elide "first" to avoid confusion
```suggestion
      // The base pointer (`rbp` in x86_64) is first register pushed onto the stack and points/references the beginning of the stack
```


nit: